cypress/integration/login.js

```diff
@@ -21,7 +21,27 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Vitalij Janukevic');
         cy.get('.main-nav').find('li').should('have.length', 1);
     })
+
+    it('Today date is highlighted in blue', function () {
+
+        cy.visit('/');
+        cy.get('[id="loginForm.userId"]').click({force:true});
+        cy.get('[aria-label="Vitalij Janukevič"]').click();
+        cy.get('[id="loginForm.role"]').click({force:true});
+        cy.get('[aria-label="User"]').click();
+        cy.get('[type="submit"]').click();
+
+        cy.url().should('include', '/time-logging');
+        cy.get('.page__title').contains('Timesheets');
+        cy.get('.calendar').should('be.visible');
+        cy.get('.tile.form').should('be.visible');
+        cy.get('.user-info__title').contains('Vitalij Janukevic');
+        cy.get('.main-nav').find('li').should('have.length', 1);
+        let currentDate = new Date();
+        cy.get('.calendar__date').contains(currentDate.getDate());

```

Comment:
I would recommend merging this test case with a previous one as almost all the steps are the same except one added line. That way you would save 10 seconds of execution time

Reply:
This ".calendar__date" should not work :) this is a cypress feature that this works, as calendar__date class have multiple occurrences, there is a specific locator for today, please use that one